requirements/base.rqr

```diff
@@ -8,3 +8,8 @@ scikit-learn
 dpath
 jiwer
 editdistance
+statsmodels
+scipy
+seaborn
+matplotlib
```

Comment:
Do we need all these requirements?  

Perhaps some can be moved to tests.rqr? Because they are only relevant if you use metric_paried_significance.

Reply:
No problem, I can move them there.  They are not only for testing applications, though, but if you think that's ok I will do it